CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ and this project adheres to
 - [#4926](https://github.com/firecracker-microvm/firecracker/pull/4926): Replace
   underlying implementation for seccompiler from in house one in favor of
   `libseccomp` which produces smaller and more optimized BPF code.
+- [#4995](https://github.com/firecracker-microvm/firecracker/pull/4995): Removed
```

Comment:
nit: I would put this under "fixed".

not nit: Let's be more specific about what exactly is changing -> what CPUID left is changing how? E.g. something like "Firecracker no longer overwrites CPUID leaf xxx when running on AMD hardware, meaning the guest will now be able to discover subleafs xxx" (and this probably depends on host kernel version?)

Reply:
Updated the entry now and moved it to fixed